README.md

```diff
@@ -17,20 +25,26 @@ This package comes from the developer of verified bots: [**@Coin Flipper#1767**]
 - [x] Detect Ghost Pings
 - [x] Send Messages As the Bot
 
-### Help
+### Help  
+
+If you don't understand something in the [**`documentation`**](https://github.com/ThatsLiamS/discord.js-ghost-ping/wiki), you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our [**`Discord Server`**](https://discord.gg/2je9aJynqt).
 
-If you don't understand something in the [documentation](https://github.com/ThatsLiamS/discord.js-ghost-ping/wiki), you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our [Discord Server](https://discord.gg/2je9aJynqt).
 
-# **Installation**
+# ❯ **Installation** 
 
-To install, do this command in your shell or command prompt then you are good to go! 
+Install with [npm](https://www.npmjs.com/) :
```

Comment:
If you could remove the space between the hyperlink `[npm](https://www.npmjs.com/)` and the colon `:` on both npm and yarn installation. With the space, the colon seems out of place (and isn't grammatically correct).

Reply:
oh..okay sure